readme.md

```diff
@@ -124,6 +124,72 @@ Also, you may be interested in adjusting Nextcloud's Datadir to store the files
+<details>
+    <summary>For a AIO setup in Docker on a Synology NAS, there are a few adjustments necessary to make it run well, especially when running the integrated reverse proxy or firewall.</summary>
+    
+    
+The main issue is that installing AIO through the **Docker GUI doesn't work**, so you need to either use SSH or create a user-defined script task in the task scheduler as the user 'root' and run it.
+Before you start make sure to create two folders on you NAS `/volume1/docker/nextcloud/data` `/volume1/docker/nextcloud/backup`
+
+The compose script without reverse proxy
+```
+sudo docker run \
+--sig-proxy=false \
+--name nextcloud-aio-mastercontainer \
+--restart always \
+--publish 80:80 \
+--publish 8080:8080 \
+--publish 8443:8443 \
+--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
+--volume /volume1/docker/docker.sock:/var/run/docker.sock:ro \
+-e NEXTCLOUD_DATADIR="/volume1/docker/nextcloud/data" \
+-e DOCKER_SOCKET_PATH="/volume1/docker/docker.sock" \
+nextcloud/all-in-one:latest 
+```
+
+ <details>
+ <summary>Setup with reverse proxy </summary>
+
+The compose script with reverse proxy which points to `localhost` at port 11000 with **HTTP** internally
+
+```
+sudo docker run \
+--sig-proxy=false \
+--name nextcloud-aio-mastercontainer \
+--restart always \
+--publish 8080:8080 \
+-e APACHE_PORT=11000 \
+-e APACHE_IP_BINDING=127.0.0.1 \
+--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
+--volume /volume1/docker/docker.sock:/var/run/docker.sock:ro \
+-e NEXTCLOUD_DATADIR="/volume1/docker/nextcloud/data" \
+-e DOCKER_SOCKET_PATH="/volume1/docker/docker.sock" \
+nextcloud/all-in-one:latest
+```
+
+![Screenshot 2023-01-20 at 11 54 41](https://user-images.githubusercontent.com/70434961/213679766-96c44fd8-12e7-4f81-9227-48476c58aaef.jpg)
+![Screenshot 2023-01-20 at 11 55 06](https://user-images.githubusercontent.com/70434961/213680608-0e11c6dc-0fa2-4961-9f55-4f7f8c977e69.jpg)
+
+- The standard backup directory doesn't work, but you can just point it to the previously created folder `/volume1/docker/nextcloud/backup`
+     
+</details>
+
+- The Synology DSM is vulnerable to attacks with it's open ports and login interfaces, which is why a firewall setup is always recommended. If a firewall is activated it is necessary to have exceptions for ports 80,443, the subnet of the docker bridge which includes the nextcloud containers (in my case 2 bridges), your public static IP (if you don't use DDNS) and if applicable your NC-Talk ports 3478 TCP+UDP:
+
+![Screenshot 2023-01-19 at 14 13 48](https://user-images.githubusercontent.com/70434961/213677995-71a9f364-e5d2-49e5-831e-4579f217c95c.png)
+    
+    
+- If you decide to run the commands through the task scheduler and want to run the command to solve the phone region error you need to remove the `-it` from it:
+
+ `sudo docker exec --user www-data nextcloud-aio-nextcloud php occ config:system:set default_phone_region --value="yourvalue" `
```

Comment:
Lets adjust this in the general docs directly and remove the -it from the command here: https://github.com/nextcloud/all-in-one#how-to-resolve-security--setup-warnings-displays-the-missing-default-phone-region-after-initial-install

```suggestion
```